velox/functions/prestosql/MapFromEntries.cpp

```diff
@@ -127,14 +128,31 @@ class MapFromEntriesFunction : public exec::VectorFunction {
       wrappedKeys = valueRowVector->childAt(0);
       wrappedValues = valueRowVector->childAt(1);
     } else {
-      wrappedKeys = decodedValueVector->wrap(
-          valueRowVector->childAt(0),
-          *inputValueVector,
-          inputValueVector->size());
-      wrappedValues = decodedValueVector->wrap(
-          valueRowVector->childAt(1),
-          *inputValueVector,
-          inputValueVector->size());
+      auto indices =
+          allocateIndices(decodedValueVector->size(), context.pool());
```

Comment:
this should handle generating indices for constant encoding, it will be inefficient but will not break. Can you confirm its handled correctly? Maybe add a test for it?

Reply:
I will address constant vector and add teat